src/highdicom/sr/utils.py

```diff
@@ -174,3 +175,35 @@ def get_coded_value(item: Dataset) -> CodedConcept:
         meaning=value.CodeMeaning,
         scheme_version=value.get('CodingSchemeVersion', None)
     )
+
+
+def get_coded_modality(item: Dataset) -> str:
```

Comment:
I am wondering whether the full dataset should be passed or just SOP Class UID value. Thoughts?